frontend/src/components/Subscribe/Subscribe.jsx

```diff
@@ -38,23 +30,35 @@ import { convertQueryStringToServerFormat } from './subscribeQueryString';
  *      A validation error for phone
  *    emailError
  *      A validation error for email
+ *    subscriptions
+ *      Newly created subscriptions (response from the server)
+ *    createSubscriptions
+ *      The function that creates a subscription (or subscriptions) on the server
+ *    loading
+ *      A boolean values that indicates whether the communication with the server is in progress
+ *    error
+ *      An error object returned from the server if there is any error
  */
 
-function Subscribe({
-  createSubscriptions,
-  isLoading,
-  error,
-  subscriptions,
-}) {
+function Subscribe() {
   const { t } = useTranslation();
 
   const history = useHistory();
   const queryString = useLocation().search;
   const [isFormSubmitted, setFormSubmitted] = useState(false);
   const [phone, setPhone] = useState('');
-  const [email, setEmail] = useState('');
+  const [email, setEmail] = useState(getUserEmail());
   const [phoneError, setPhoneError] = useState(null);
   const [emailError, setEmailError] = useState(null);
+  const [subscriptions, setSubscriptions] = useState(null);
+
+  const [createSubscriptions, { loading, error }] = useMutation(
+    CREATE_SUBSCRIPTIONS,
+    {
+      onCompleted: data => data && data.createSubscriptions ?
+        setSubscriptions(data.createSubscriptions) : setSubscriptions(null)
+    }
```

Comment:
Can be simplified to:
```suggestion
    {
      onCompleted: data => setSubscriptions(data?.createSubscriptions ?? null)
    }
```

Reply:
Oh, thank you @terchiem, it's nice. I never used this operator before. Has it been added recently or did I just miss it?

Reply:
It's pretty new, I think it was added last year.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator